examples/sample-app-ts/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "baseUrl": "./",
+        "outDir": "dist",
+        "declaration": true,
+        "declarationMap": true,
+        "module": "ES2015",
+        "moduleResolution": "node",
+        "target": "ES2015",
+        "strict": true,
+    },
+    // NOTE: `test/**/*` is automatically included in `blockly-scripts start`.
```

Comment:
The comment is definitely wrong (we aren't using `blockly-scripts start`), but I think this entire property needs to go as currently there is no `test` directory. Eventually it should have tests, but if the tests are in ts too then i'm not sure if we would want this or not.

Reply:
Good catch! Fixed.